jekyll/_cci2/first-steps.adoc

```diff
@@ -10,62 +10,119 @@ contentTags:
 :icons: font
 :experimental:
 
-To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your GitHub, Bitbucket or GitLab account, or your email address for the option to connect to your code later in the process.
+//TODO: Check backlinks
 
-[#vcs-signup]
-== Sign up with GitHub or Bitbucket
+To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your email address and connect your GitHub, Bitbucket, or GitLab accounts in the process.
+
+- If you are **creating a new CircleCI org** for your team, follow the xref:#sign-up-and-create-an-org[Sign up and create an org] steps below. 
+- If you are signing up for a CircleCI account through an **invite** from someone on your team, follow the steps in the xref:#sign-up-with-an-invite[Sign up with an invite] section.
+
+[#vcs-integrations]
+== VCS integrations
+
+CircleCI supports GitHub, Bitbucket, and GitLab projects. If you use **GitHub** or **GitLab**, and are setting up a CircleCI organization for your team, you will be setting the VCS integration type and authenticating with your VCS provider during the signup process. Team members that you invite afterwards to your CircleCI org will have to also later authenticate with the VCS provider if they are to create new projects.
```

Comment:
I had to read this section a few times to work out what the difference is. Maybe the language can be simplified a bit?

Reply:
Ah, I think I was trying to allude to who's required to do an OAuth handshake and when as described here https://circleci.atlassian.net/wiki/spaces/PES/pages/6891044888/GitHub+App+User+Flows#New-user-joining-a-GitHub-App-Org I'll think on rephrasing this!

Reply:
Yeah it does totally make sense what you have I just took a few goes to get the difference between how it works for standalone and non-standalone I guess.